packages/webpack-config/src/webpack/base/legacy.js

```diff
@@ -11,6 +11,9 @@ import { compileLitHTML } from "./tasks/compileLitHTML";
 import { loadFonts } from "./tasks/loadFonts";
 import { compileShadowCSS } from "./tasks/compileShadowCSS";
 import { loadHandlebars } from "./tasks/loadHandlebars";
+import { copyStatic } from "./tasks/copyStatic";
```

Comment:
it's enough to start this copy task once - best place is "module.js".
Copytask should be removed here.

Reply:
done